include/cpu.h

```diff
@@ -326,7 +326,7 @@ struct TSS_32 {
 class Descriptor
 {
 public:
-	Descriptor() { saved.fill[0]=saved.fill[1]=0; }
+	Descriptor() : saved{{0, 0}} {}
 
```

Comment:
Nice cleanup.  More eye-strain on that original code!

Reply:
Figuring out how to initialize union in a C++ initializer list was a little bit tricky actually ;)